MLDSAI/OpenAdapt
# Automatic Publishing with PyPi

## Overview

- Resolves #22 
- Using Github Actions script that automatically publishes when a new release is created

## Unit Tests Created

- n/a

## Steps to QA

- See Github Actions run history on my repo: https://github.com/Mustaballer/puterbot/actions
- Check out: https://test.pypi.org/project/openadapt/0.0.1/
